txcache/selectionTracker_test.go

```diff
@@ -16,6 +16,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var defaultSelectionSessionMock = txcachemocks.SelectionSessionMock{
```

Comment:
It still occupies the global namespace, and has the "default" prefix. I meant move within the test itself.